orttraining/orttraining/core/session/training_session.cc

```diff
@@ -514,7 +514,7 @@ static Status AddGradientAccumulationNodes(Graph& graph,
 
 Status TrainingSession::ApplyTransformationsToMainGraph(const std::unordered_set<std::string>& weights_to_train,
                                                         const TrainingConfiguration::GraphTransformerConfiguration& config) {
-  GraphTransformerManager graph_transformation_mgr{1};
+  GraphTransformerManager graph_transformation_mgr{2};
```

Comment:
is this necessary for the fusion to take place? would probably be good to validate the e2e tests pass with this.

Reply:
It isn't required. But > 0 is more consistent with the training_runner.cc default (=5) or session options default (=10).

Here's a passing e2e pipeline test:
https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=133280&view=results

Reply:
I suppose this is more appropriate:
https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=133304&view=results